ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/tunnel-interfaces.j2

```diff
@@ -0,0 +1,65 @@
+{# eos - Tunnel Interfaces #}
+{% for tunnel_interface in tunnel_interfaces | arista.avd.natural_sort('name') %}
+!
+interface {{ tunnel_interface.name }}
+{%     if tunnel_interface.description is arista.avd.defined %}
+   description {{ tunnel_interface.description }}
+{%     endif %}
+{%     if tunnel_interface.shutdown is arista.avd.defined(true) %}
+   shutdown
+{%     elif tunnel_interface.shutdown is arista.avd.defined(false) %}
+   no shutdown
+{%     endif %}
+{%     if tunnel_interface.mtu is arista.avd.defined %}
+   mtu {{ tunnel_interface.mtu }}
+{%     endif %}
+{%     if tunnel_interface.vrf is arista.avd.defined and tunnel_interface.vrf is ne 'default' %}
```

Comment:
according to contribution guide we use `!=` and `==`

Reply:
this was taken from `templates/eos/management-interfaces.j2` should I clean it as well?

Reply:
Addressed in 0db9c4a